packages/enzyme-context-apollo-3/src/index.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  ApolloClient,
+  ApolloProvider,
+  InMemoryCache,
+  PossibleTypesMap,
+  NormalizedCacheObject,
+} from '@apollo/client';
+import { SchemaLink } from '@apollo/client/link/schema';
+import React from 'react';
+import { EnzymePlugin, composeWrappingComponents } from 'enzyme-context-utils';
+import {
+  makeExecutableSchema,
+  IExecutableSchemaDefinition,
+  IMocks,
+  addMockFunctionsToSchema,
+} from 'graphql-tools';
+import { defaultMocks } from './Utils';
+
+export type ApolloPluginConfig = {
+  possibleTypes?: PossibleTypesMap;
+  schema: IExecutableSchemaDefinition;
+  defaultMocks?: IMocks;
+};
```

Comment:
Can we also add the ability to pass `defaultOptions` through to the `ApolloClient`?

Reply:
Sure, I'll do that in a followup